modules/develop/partials/run-transforms.adoc

```diff
@@ -99,11 +99,17 @@ See also: xref:./how-transforms-work.adoc[]
 . Create and initialize a data transforms project:
```

Comment:
Does it make sense to have a single tab for this whole section instead of two tabs? I don't know how that looks, but something to consider.

Reply:
Thanks I really actually like the single tab here!